sites/public/styles/overrides.scss

```diff
@@ -14,43 +17,105 @@
     border: none;
 
     .navbar {
+      align-items: center;
       height: auto;
       justify-content: flex-start;
       max-height: none;
-      padding: 8px 0 16px;
+      padding: 24px 0 18px;
+
+      @media (min-width: $header-desktop-min-size) {
+        padding: 0 12px;
+        width: 100%;
+      }
 
-      @screen md {
-        padding: 0;
+      @screen lg {
+        padding: 0 50px;
+        width: 100%;
       }
 
       @screen xl {
         justify-content: center;
       }
 
       .navbar-menu {
-        height: 80px;
-        min-width: 112px;
+        height: 48px;
+        margin-right: 12px;
+        min-width: 48px;
+        width: 48px;
 
-        @screen md {
+        @media (min-width: $header-desktop-min-size) {
           margin: 40px 0 24px;
-          position: absolute;
-          right: 1rem;
+          width: 100%;
         }
 
         @screen xl {
           margin: 48px 0 32px;
         }
 
+        .button.navbar-mobile-menu-button {
+          align-items: center;
+          justify-content: center;
+          margin: 0;
+          padding: 0;
+        }
+
+        .button {
+          width: 60px;
+        }
+
+        .button__content {
+          display: none;
+        }
+
+        .button__icon {
+          margin-left: 0;
+        }
+
+        .ui-base svg {
+          height: 18px;
+          width: 21px;
+        }
+
         .navbar-link {
-          @apply text-sm;
           @apply text-primary;
           align-items: center;
           display: flex;
+          font-size: 1.125rem;
           font-weight: 700;
+          height: 40px;
           padding: 1rem;
           text-transform: none;
         }
 
+        .navbar-link__sign-in {
+          margin-left: 1rem;
+          padding-left: 2rem;
+          position: relative;
+        }
+
+        // This is a little hacky to get the divider line to the left of the "sign in" link, but
+        // this is the simplest way to do it without making significant changes to the SiteHeader
+        // component
+        .navbar-link__sign-in:before {
+          border-left: 3px solid $tailwind-primary;
```

Comment:
Does this work in Arabic?

Reply:
Fixed! I was going to check this but I forgot, so thanks for calling this out! 

![image](https://user-images.githubusercontent.com/371177/145655446-ac7003c8-6508-450f-b764-7f575902e847.png)